tools/src/32Blit.cpp

```diff
@@ -299,7 +299,7 @@ std::string GuessPortName()
     return "";
 
   io_object_t portService;
-  char devicePath[1024]{};
+  char devicePath[1024];
```

Comment:
`= {}`? or at least zero it somehow, otherwise the return value will be uninitialised when no port is found.